packages/pwa-kit-react-sdk/scripts/build.js

```diff
@@ -36,7 +36,7 @@ const transpileSrc = () => {
     console.log(`Transpiling files in src/ to dist/`)
 
     const child = exec(
-        `npx --no-install babel --root-mode upward src -x ".js",".jsx" --ignore "**/test_fixtures/*","*.test.js","test.js" --out-dir dist`
+        `npx --no-install babel --root-mode upward src -x ".js",".jsx" --ignore "**/test_fixtures/*","*.test.js","test.js" --out-dir dist --copy-files`
```

Comment:
What is this addition of the `--copy-files` argument about?

Reply:
The babel settings we have doesn't include the json files in `dist`, this flag will copy non-js files and make sure they are in `dist`. This way people can import the `schema.json` from the SDK. But since we just moved the validation logic from pwa to the sdk, this flag is not a must have anymore.

Do you think it's necessary `schema.json` from the SDK?